FFXIVClientStructs/FFXIV/Component/GUI/AtkTextNode.cs

```diff
@@ -128,3 +126,33 @@ public enum FontType : byte {
     Jupiter = 0x4,
     JupiterLarge = 0x5,
 }
+
+[StructLayout(LayoutKind.Explicit, Size = 0x38)]
+public unsafe struct LinkData {
+    [FieldOffset(0x00)] public int Unk0;
+    [FieldOffset(0x04)] public int Unk4;
+    [FieldOffset(0x08)] public int Unk8;
+    [FieldOffset(0x0C)] public int UnkC;
+    /// <remarks> To be read until link terminator. </remarks>
+    [FieldOffset(0x10)] public byte* Payload;
+    /// <remarks> First 3 bytes are PayloadEnd(?) and last byte is <see cref="LinkType"/> </remarks>
+    [FieldOffset(0x18), CExportIgnore] public uint PayloadEndAndLinkType;
```

Comment:
Is this offset set as an unsigned int somewhere?

Reply:
Well, it contains 2 values, so it wouldn't make sense if it's int, because one bit is used for the sign then.
It converts it after bitshifting to an int, but it should be uint before that, or you get wrong values, I think?
I wrote the sigs in the original message where you can find it.

Reply:
And coverting it to an int after bitshifting also doesn't make sense, becaue you'd have the sign at the wrong position. But that's what IDA shows. Not sure if accuate.